cuda_bindings/cuda/bindings/path_finder.py

```diff
@@ -1,13 +1,47 @@
 # Copyright 2024-2025 NVIDIA Corporation.  All rights reserved.
-#
 # SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
 
-from cuda.bindings._path_finder.load_nvidia_dynamic_library import (
-    load_nvidia_dynamic_library as _load_nvidia_dynamic_library,
-)
-from cuda.bindings._path_finder.supported_libs import SUPPORTED_LIBNAMES as _SUPPORTED_LIBNAMES
+from cuda.pathfinder import SUPPORTED_NVIDIA_LIBNAMES as _SUPPORTED_LIBNAMES
+from cuda.pathfinder import load_nvidia_dynamic_lib
 
 __all__ = [
     "_load_nvidia_dynamic_library",
     "_SUPPORTED_LIBNAMES",
 ]
```

Comment:
Given this module is effectively part of the public API, is using `__all__` here sufficient to guard against the non-underscored import names and class we're introducing?

Reply:
I do not think it's fine. We're following strict semver here, which means if you can publicly access something via `cuda.bindings.path_finder` that doesn't begin with an underscore that it is part of our API and we will not break it in bumping minor versions.

If this is only a temporary stopgap that we don't intend to survive until a release then that is fine.

Reply:
I've added commit 58ca181684671988101dcac68fa7b7d3dfc527aa, which fully sanitizes the `cuda/bindings/path_finder.py` module namespace. Retesting.